src/Http/Error.php

```diff
@@ -45,13 +45,33 @@ public function __construct(
         string $message = ''
     ) {
 
+        $factory = \NDB_Factory::singleton();
+
         $uri     = $request->getURI();
         $baseurl = $uri->getScheme() .'://'. $uri->getAuthority();
-
+        // Admistrator email.
+        $contact  = $request
+            ->getAttribute('loris')
+            ->getConfiguration()
+            ->getSetting('mail')['From'];
```

Comment:
why don't you just pass in the administrator email and the user object directly?

Reply:
Wouldn't that require updating all of the calling code?

Reply:
I didn't realize $request was already being passed, I thought you added it for this and were changing it anyways.. nevermind, since it's already there you can use it directly